dll/win32/syssetup/lang/fr-FR.rc

```diff
@@ -213,7 +213,7 @@ BEGIN
     IDS_WZD_NAME "L'installation ne peut pas continuer tant que vous n'avez pas entré votre nom."
     IDS_WZD_SETCOMPUTERNAME "L'installation a échoué lors de la définition du nom de votre ordinateur."
     IDS_WZD_COMPUTERNAME "L'installation ne peut continuer tant que vous n'avez pas entré le nom de votre ordinateur."
-    IDS_WZD_PASSWORDEMPTY "Vous devez entrer un mot de passe !"
+    IDS_WZD_PASSWORDEMPTY "Vous devez entrer un mot de passe!"
```

Comment:
No, french needs a space.

Reply:
No space afaik. Check rules

Reply:
I asked @KRosUser2 who is a native french speaker, and he confirmed that no space would be needed before ! in french.
Only a space before a ":" character is required for french.

Reply:
That being said, ofc I will happily apply the way for french where majority of french native speakers would feel most comfortable with.

Reply:
> No space afaik. Check rules ("espace insécable")

Where are these rules defined/explained?

Check, for example,
https://storylearning.com/learn/french/french-tips/french-punctuation

Reply:
Guys you're all wrong (except Serge). French puts a space between interrogation point, exclamation point, double-point (:) and semi-colon (;) - The list given by the last link is correct by the way.

Reply:
Okay, "2 vs 1 of native french speakers" voted for changing back the space for french. I will do so therefore.